community/git.md

```diff
@@ -0,0 +1,19 @@
+# Sharing mods with git
```

Comment:
Why a separate file? (Just asking)

Reply:
To encourage the use of git.
And it is different than normal sharing.
I could change the title slightly.

Reply:
Since the other file is called ‘sharing’ as if it included everything, it seems more like discouraging to me.

Reply:
So you want me to put it all in one file? I was going to link to the Git guide. I think it is a bit more advanced, thus the separate guide.

Reply:
Prolly yes. What do you think, @ortiza5?

Reply:
What if I have a section in sharing.md about git and the benefits of git, and then link to the git guide to show how to actually do it and give examples of other people’s repos.

Reply:
> What if I have a section in sharing.md about git and the benefits of git, and then link to the git guide to show how to actually do it

I’d prefer the details to be part of a guide about managing mods, so you don’t need to add anything not related to sharing.

> and give examples of other people’s repos.

I’d prefer these to be in a *useful links/resources* file (sorry, forgot about it before)

Reply:
Could we make the GitHub wiki useful links/resources for now? I don’t want to start pull requests just to add a single link.

Reply:
I don’t see why. The most important ones will be added in one PR, then you will submit a PR with just the one link you want to add. I also feel bad when I do almost no changes in a PR, but it’s really OK.

Reply:
It just feels useless to save all my work, go into GitHub desktop, make a new branch, stash changes, switch to branch, edit a single file, make a pull request, explain pull request, exit branch, every time I want to add a link.

On the other hand, we could leave a wiki page open for editing to collect links, and then add them as a file when we’re ready.

Reply:
Using the wiki means having things in two places, & it’s never gonna be ready for the ‘final state’.

> save all my work, go into GitHub desktop, make a new branch, stash changes, switch to branch, edit a single file, make a pull request, explain pull request, exit branch

Come on, that’s nothing hard, & if you’re gonna do it very often, you can wait til you have several links. Or you can do it from the GH web interface, which doesn’t interfere w/ your local index.